### ISSUE TITLE
More unlabeled buttons with TalkBack.

### ISSUE THREAD

**Issue by shortwavesurfer2009** [hjiangsu/thunder#1787]
### Bug Description

When using the TalkBack screen reader, I found some more unlabeled buttons.

![Image](https://github.com/user-attachments/assets/6acdbbb7-2569-41f2-929d-2112f86a74c8)

### Expected Behaviour

The buttons should be properly labeled.

### Steps to Reproduce

Enable TalkBack from the Android system settings and tap the buttons shown in the screenshot.

### App Version

0.7.0-1

### Device

Oneplus nord n200

### OS

android 14
